src/Validator/Constraint/Address.php

```diff
@@ -29,15 +29,9 @@ class Address extends Constraint
         self::INVALID_ADDRESS_ERROR => 'INVALID_ADDRESS_ERROR',
     ];
 
-    /**
-     * @var string
-     */
-    public $service = AddressValidator::class;
+    public string $service = AddressValidator::class;
 
-    /**
-     * @var string
-     */
-    public $message = 'Address {{ address }} is not valid.';
+    public string $message = 'Address {{ address }} is not valid.';
```

Comment:
I'd move these as promoted properties. And options should be removed ant `null` should be passed to parent constructor :)